src/coreclr/components.cmake

```diff
@@ -19,3 +19,4 @@ add_dependencies(runtime jit iltools)
 
 # The cross-components build is separate, so we don't need to add a dependency on coreclr_misc
 add_component(crosscomponents)
+add_dependencies(crosscomponents jit)
```

Comment:
Adding this dependency breaks the crossdac build, which is considered a crosscomponent. It also tries to run shell scripts on linux when running `./src/coreclr/build-runtime.cmd checked x64 -linuxdac`. We'd need to exclude building the JIT under the crossdac build.